src/search/components/dateTime.component.tsx

```diff
@@ -295,6 +295,9 @@ const DateTimeSearch = (props: DateTimeSearchProps): React.ReactElement => {
             <DateTimePicker
               format="yyyy-MM-dd HH:mm"
               value={datePickerFromDate}
+              referenceDate={
+                datePickerFromDate || datePickerToDate || new Date()
+              }
```

Comment:
This changes means that if the fromDate is null that the date picker is opened on the ToDate, if that is also null then it gets opened on the current date

Reply:
Users are still are still able to navigate to today's date by clicking the today button in the date picker